sys/include/net/credman.h

```diff
@@ -185,6 +186,76 @@ void credman_delete(credman_tag_t tag, credman_type_t type);
  */
 int credman_get_used_count(void);
 
+#if IS_USED(MODULE_CREDMAN_LOAD) || DOXYGEN
+/**
+ * @brief Load a public key from a buffer, as a `SubjectPublicKeyInfo` sequence, according to
+ *        RFC5280. The key should be encoded in DER format.
+ *
+ * @pre `buf != NULL && out != NULL`.
+ *
+ * @note To use this functionality include the module `credman_load`. Credman only supports ECDSA
+ *       for now, so [RFC5480](https://tools.ietf.org/html/rfc5480) applies.
+ *
+ * @experimental This API is considered experimental and will probably change without notice!
+ *
+ * @see https://tools.ietf.org/html/rfc5280#section-4.1
+ *
+ * @param[in]  buf          Buffer holding the encoded public key
+ * @param[in]  buf_len      Length of @p buf
+ * @param[out] out          ECDSA public key to populate
+ *
+ * @retval CREDMAN_OK on success
+ * @retval CREDMAN_INVALID if the key is not valid
+ */
+int credman_load_public_key(const uint8_t *buf, size_t buf_len, ecdsa_public_key_t *out);
```

Comment:
Would it make sense to use
```suggestion
int credman_load_public_key(const void *buf, size_t buf_len, ecdsa_public_key_t *out);
```
here?

Reply:
I think it could make sense, [`credman_buffer_t`](http://doc.riot-os.org/structcredman__buffer__t.html) is already using `void *` so it kind of fits.

Reply:
Changed for all